AppImageBuilder.yml

```diff
@@ -0,0 +1,63 @@
+version: 1
+
+AppDir:
+  path: ./AppDir
+
+  app_info:
+    id: minetest
+    name: Minetest
+    icon: minetest
+    version: !ENV ${VERSION}
+    exec: usr/bin/minetest
+    exec_args: $@
+  runtime:
+    env:
+      APPDIR_LIBRARY_PATH: $APPDIR/usr/lib/x86_64-linux-gnu
+
+  apt:
+    arch: amd64
+    sources:
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ bionic main universe
+        key_url: 'http://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3b4fe6acc0b21f32'
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ bionic-updates main universe
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ bionic-backports main universe
+      - sourceline: deb http://archive.ubuntu.com/ubuntu/ bionic-security main universe
+
+# Included stuff: libirrlicht-dev libbz2-dev libpng-dev libjpeg-dev libxxf86vm-dev libgl1-mesa-dev libsqlite3-dev libogg-dev libvorbis-dev libopenal-dev libcurl4-gnutls-dev libfreetype6-dev zlib1g-dev libgmp-dev libjsoncpp-dev
+    include:
+      - libirrlicht1.8
+      - libbz2-1.0
+      - libpng16-16
+      - libjpeg8
```

Comment:
transitive irricht deps, not needed here

Reply:
Changed, thanks!